### PR TITLE
Created a separate FeatureToggle class with the northboud calls

### PR DESCRIPTION
**Pull request by izadorozhna**
Implements #5560

* Created a separate FeatureToggle class with the methods that call northboud API calls
* Applied new methods in the toggles specs
* Applied new approach in the other specs

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy [hunk 1]
```diff
@@ -663,6 +665,10 @@ triggering one more reroute of the current path"
 @Isolated
 class AutoRerouteIsolatedSpec extends HealthCheckSpecification {
     //isolation: global toggle flowsRerouteOnIslDiscoveryEnabled is changed
+    @Autowired
+    @Shared
+    FeatureToggles featureToggles
```
[the remark this answers is not in the document]
Comment (izadorozhna) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy]: Fixed and pushed the new PR version, thank you!

File: src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FeatureToggles.groovy [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Component
+
+@Slf4j
+@Component
+class FeatureToggles {
+
+    @Autowired @Qualifier("islandNbV2")
+    NorthboundServiceV2 northboundV2
```
Comment (yuliiamir) [inline src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FeatureToggles.groovy:16]: Do we need this var?

Comment (izadorozhna) [reply src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FeatureToggles.groovy:16]: In general, no, removed it

File: src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FeatureToggles.groovy [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+    @Autowired @Qualifier("islandNb")
+    NorthboundService northbound
+
+    private static FeatureToggles ftInstance
```
[the remark this answers is not in the document]
Comment (izadorozhna) [reply src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FeatureToggles.groovy:21]: The `ftInstance` was created by me due to the idea of a singleton. So I wanted to create the only one instance of the class

**Pull request merged**